src/components/SWActivationCard/SWActivationCard.js

```diff
@@ -36,6 +50,38 @@ class SWActivationCard extends React.Component<Props, State> {
     isModalVisible: false,
   }
 
+  getModalItems = () => {
+    const {
+      navigation, baseFiatCurrency, balances, rates,
+    } = this.props;
+    const fiatCurrency = baseFiatCurrency || defaultFiatCurrency;
+    const ethBalance = getBalance(balances, ETH);
+    const balanceInFiat = ethBalance * getRate(rates, ETH, fiatCurrency);
+    const fiatAmount = formatFiat(balanceInFiat, baseFiatCurrency || defaultFiatCurrency);
+
+    return [
+      {
+        label: 'I have ETH',
+        money: fiatAmount,
+        onPress: () => {
+          navigation.navigate(CHOOSE_ASSETS_TO_TRANSFER);
+        },
+        key: 'has ETH',
+      },
+      {
+        label: "I'd like to have some",
+        chevron: true,
+        onPress: () => {
+          navigation.navigate(EXCHANGE, {
+            fromAssetCode: fiatCurrency,
+            toAssetCode: 'ETH',
```

Comment:
There's ETH as a constant in assetsConstants (and it is already imported in this doc), it would be better to use it here.